### DISCUSSION TITLE
CanCreateDirectories for linux and windows

### DISCUSSION THREAD

**Discussion by emm1R** [wailsapp/wails#2417] [Q&A]
Hi, is there any plans on enabling Linux and Windows to create directories when the wails dialogs are in use?

**Comment by leaanthony** [comment]
Does it not work by default?

**Reply by emm1R** [reply]
Oh, I just read the documents and thought that it does not work. Now that I tested it is does. Whoops.

**Reply by leaanthony** [reply]
Glad it's working. Sounds like the docs could be clearer. Would you be happy to do a PR for that?
